text/skia_text_blob_shaper.cpp

```diff
@@ -34,7 +34,8 @@ class ShaperRunHandler final : public SkShaper::RunHandler {
                    const gfx::PointF& offset,
                    TextBlob::RunHandler* subHandler)
     : m_builder(utf8Text, os::to_skia(offset))
-    , m_subHandler(subHandler) { }
+    , m_subHandler(subHandler)
+    , m_buffer() { }
```

Comment:
The `m_buffer` is a `Buffer` struct without constructor and pointers, is this the line that fixed the crash? (with this change we're initializing `m_buffer`'s internal pointers to `nullptr`)

Reply:
No, this was just a compiler suggestion and I tend to prefer initializing everything, but I don't think it changes anything

Reply:
Actually this initialization does initialize the `m_buffer` members to `nullptr`, I've tested that: https://godbolt.org/z/9s5YjETWP

Without this the m_buffer member pointers are not set to `nullptr` (that's because `Buffer` struct does't have a constructor).